Gems/Blast/Code/Source/Components/BlastFamilyComponent.h

```diff
@@ -34,8 +34,8 @@ namespace Blast
     //! Component that handles simulation of the Blast family.
     class BlastFamilyComponent
         : public AZ::Component
-        , public BlastFamilyDamageRequestBus::MultiHandler
-        , public BlastFamilyComponentRequestBus::Handler
+        , protected BlastFamilyDamageRequestBus::MultiHandler
+        , protected BlastFamilyComponentRequestBus::Handler
```

Comment:
question: For request buses having them as public usually isn't a bad idea (unlike notification buses that should usually be protected or private) Is there any reason in particular for this change?

Reply:
This is to avoid anybody from calling the request functions directly, and force them to use the buses instead. This is important in this specific case because the functions shouldn't be called unless `Spawn` is called (which is done in Activation only and only then it connects to the request buses). If the requests functions would be public that would mean that anybody could call them at anytime and protection code would have to be added to all the request functions. So it's better to protect them and avoid that from happening.

Reply:
Ah okay understood, thanks for the explanation :+1: